src/plugins/expressions/common/expression_types/specs/error.ts

```diff
@@ -22,8 +22,14 @@ export type ExpressionValueError = ExpressionValueBoxed<
   }
 >;
 
-export const isExpressionValueError = (value: unknown): value is ExpressionValueError =>
-  getType(value) === 'error';
+export const isExpressionValueError = (value: unknown): value is ExpressionValueError => {
```

Comment:
type guard method should returns only `true/false `without triggering an exception